networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,83 @@ These release notes track the development of the Network Observability Operator
+
+[id="network-observability-without-loki-1.4"]
+==== Network Observability without Loki
+The Network Observability Operator is now functional and usable without Loki. If Loki is not installed, it can only export flows to KAFKA or IPFIX format and provide metrics in the Network Observability metrics dashboards. For more information, see xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki]. 
+
+[id="network-observability-enhanced-configuration-and-ui-1.4"]
+==== Notable enhancements
+The 1.4 release of the Network Observability Operator adds improvements and new capabilities to the {product-title} web console plugin and the Operator configuration. 
+
+This release introduces the following web console enhancements:
+
+* In the *Query Options*, the *Duplicate flows* checkbox is added to choose whether or not to show duplicated flows.
+* You can now filter source and destination traffic with image:arrow-up-long-solid.png[,10] *One-way*, image:arrow-up-long-solid.png[,10] image:arrow-down-long-solid.png[,10] *Back-and-forth*, and *Swap* filters. 
+* The Network Observability metrics dashboards in *Observe* -> *Dashboards* -> *NetObserv* and *NetObserv / Health* are modified as follows:
+** The *NetObserv* dashboard shows top bytes, packets sent, packets received per nodes, namespaces, and workloads. Flow graphs are removed from this dashboard.
+** The *NetObserv / Health* dashboard shows flows overhead as well as top flow rates per nodes, namespaces, and workloads.
+** Infrastructure and Application metrics are shown in a split-view for namespaces and workloads. 
+  
+
+For more information, see xref:../../networking/network_observability/observing-network-traffic.adoc#network-observability-quickfilter_nw-observe-network-traffic[Configuring Quick Filters] and xref:../../networking/network_observability/network-observability-overview.adoc#network-observability-dashboards[Network Observability metrics].
+ 
+This release introduces the following configuration enhancements:
+
+* You now have the option to specify different namespaces for any configured ConfigMap or Secret reference, such as in certificates configuration.
+
+For more information, see xref:../../networking/network_observability/configuring-operator.adoc#network-observability-flowcollector-view_network_observability[Flow Collector sample resource] and xref:../../networking/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference].
+
+[id="network-observability-dns-tracking-1.4"]
+==== DNS tracking
+In 1.4, the Network Observability Operator makes use of eBPF tracepoint hooks to enable DNS tracking DNS tracking. You can monitor your network, conduct security analysis, and troubleshoot DNS issues in the *Network Traffic* and *Overview* views in the web console. 
```

Comment:
```suggestion
In 1.4, the Network Observability Operator makes use of eBPF tracepoint hooks to enable DNS tracking. You can monitor your network, conduct security analysis, and troubleshoot DNS issues in the *Network Traffic* and *Overview* views in the web console. 
```

Reply:
Copy/paste error on my part. I _just_ reworded this and left the duplicate DNS tracking 🤦‍♀️